src/js/bootstrap-slider.js

```diff
@@ -1214,6 +1250,23 @@ const windowIsDefined = (typeof window === "object");
 			        }
 				}
 			},
+			_createHighlightRange: function (start, end) {
+				if (this._isHighlightRange(start, end)) {
+					if (start > end) {
+						return {'start': end, 'size': start - end};
+					}
+					return {'start': start, 'size': end - start};
+				}
+				return null;
+			},
+			_isHighlightRange: function (start, end) {
+				if (0 <= start && start <= 100) {
```

Comment:
since the `if` clause only returns a different input of the inner-most clause is true, could you hoist the inner `if` clause up so that we can avoid nested `if...else`?

```js
if (0 <= start && start <= 100 && 0 <= end && end <= 100) {
     return true;
}
else {
     return false;
}
```